doc/DEMO.md

```diff
@@ -0,0 +1,624 @@
+```
+cargo run -- --conf cosigner_1_config.toml
+cargo run -- --conf cosigner_2_config.toml
+```
+
+Go back to the parent directory for the next step
+```
+cd ..
+```
+
+### 7. Updating the coordinatord config
+
+`coordinatord` needs all the Noise keys in its config.
+
+```
+cd coordinatord
+```
+
+Open the configuration file in `coordinatord_config.toml`, add all the manager noise keys and all the stakeholders noise keys. Leave the watchtowers keys empty, they're not used yet.
+
+Your config file now should look like this:
+```
+managers = [
+    "d2deeb8398f47789e1f5118c42834031e3722817a432192b74c363fdb36cc634",
+]
+
+stakeholders = [
+    "eecd2a93f5b09b88519f38d620aa127333d1934987d18001132f07ffa3596c65",
+    "614ad96890c309b8da6915ddb9eb1135caf228120178833e70798f20e0783b16",
+]
+
+watchtowers = []
+```
+
+Now start the `coordinatord` again:
+
+```
+cargo run -- --config coordinatord_config.toml
+```
+
+Go back to the parent directory for the next step
+```
+cd ../../
+```
+
+### 8. Sending commands using the cli
+Let's go back to the revaultd directory and start everything once again:
+```
+cd revaultd
+stake_1_d
+stake_2_d
+man_1_d
+```
+
+Now, let's ask for info:
+
+```
+stake_1_cli getinfo
+```
+
+You should see something like:
+```
+{
+  "result":{
+    "blockheight":215,
+    "network":"regtest",
+    "sync":1.0,
+    "vaults":2,
+    "version":"0.0.2"
+  }
+}
+```
+
+Do the same with `stake_2_cli` and `man_1_cli`, just to make sure everything works :)
+
+### 9. Checking if you fucked up
+
+Please run
+
+```
+stake_1_cli getdepositaddress
+```
+
+and then
+
+```
+stake_2_cli getdepositaddress
+```
+
+and then
+
+```
+man_1_cli getdepositaddress
+```
+
+If they all return the same address, cool, somehow you managed to not fuck it up. Otherwise, well, you fucked up.
+Don't worrry, it happens. We're working towards making it happen less often by introducing descriptors in the configuration file, but in the meanwhile, you'll have to debug.
+
+If the deposit address are different, it means that the descriptors created by `revaultd` are different. Please check that:
+
+- The `stakeholder_xpubs` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `manager_xpubs` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `cosigner_keys` are all equal in all configurations, **the order of the keys MUST be the same**
+- The `unvault_csv` is the same in every configuration
+
+Delete your sqlite database and your watchonly wallet:
+
+```
+rm ./stake_1_data/revaultd.sqlite3
+rm ./stake_2_data/revaultd.sqlite3
+rm ./man_1_data/revaultd.sqlite3
+rm -r ./stake_1_data/revaultd-watchonly-wallet-1/
+rm -r ./stake_2_data/revaultd-watchonly-wallet-1/
+rm -r ./man_1_data/revaultd-watchonly-wallet-1/
+```
+
+Restart `revaultd` and check again the deposit addresses. If that didn't work either, try restarting `bitcoind`.
+
+### 10. Wait, CLI only?!
```

Comment:
Just before maybe point them to the list of available commands in `doc/API.md` ?